### PR TITLE
Updates in prepartion for release v2019.2

### PR DESCRIPTION
**Pull request by matsduf**
Updates for v2019.2 release:

* Changes file with all updates since last release
* Updated version (to v2.1.0) in `lib/Zonemaster/LDNS.pm`

### CODE REVIEW

**Comment by matsduf** [comment]
Will need to be updated when/if #92 is merge.

File: Changes [hunk 1]
```diff
@@ -1,13 +1,24 @@
 Release history for Zonemaster component Zonemaster-LDNS
 
+
+3.0.0 2019-11-15
+
+ [API change]
```
Comment (mattias-p) [inline Changes]: What's the thought behind the change in format? Is it to comply with [CPAN::Changes::Spec](https://metacpan.org/pod/distribution/CPAN-Changes/lib/CPAN/Changes/Spec.pod)? Maybe we should mention that in the Versions and Releases document?

File: Changes [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+ - Support for Ed25519 (algorithm 15) added (#85, #51, #84)
```
Comment (mattias-p) [inline Changes]: This should be labelled as an addition rather than a change. Or maybe I'm missing something?

File: Changes [hunk 3]
```diff
@@ -1,13 +1,24 @@
...
+ - Requires libldns 1.7.0 or 1.7.1 (#85)
+
+ [Fixed]
+ - Make Travis not fail (#82)
```
Comment (mattias-p) [inline Changes]: This isn't a change to the software. I don't think it belongs in this change log.

**Comment by mattias-p** [comment]
By the way I think this too is a pretty good guideline: https://keepachangelog.com/en/1.0.0/

File: Changes (hunk 1 above)
Comment (matsduf) [reply Changes]: Good idea to add the reference to documentation. Will look at that.

File: Changes (hunk 2 above)
Comment (matsduf) [reply Changes]: Well, an addition is a change. I changed to "Changed".

File: Changes (hunk 3 above)
Comment (matsduf) [reply Changes]: I agree that it is a border case. I see no problem of keeping it.

File: Changes (hunk 2 above)
Comment (mattias-p) [reply Changes]: Not in the semver sense. In this context "change" means breaking change and "addition" means it's backwards compatible. The addition of Ed25519 is backwards compatible, but the version bump of libldns is not.

File: Changes [hunk 4]
```diff
@@ -3,10 +3,14 @@ Release history for Zonemaster component Zonemaster-LDNS
 
 3.0.0 2019-11-15
 
- [Changed]
- - Support for Ed25519 (algorithm 15) added (#85, #51, #84)
+ [API Change]
```
Comment (mattias-p) [inline Changes]: How are these API changes?

File: Changes [hunk 5]
```diff
@@ -1,13 +1,28 @@
...
 
+
+3.0.0 2019-11-15
+
+ [API Change]
+ - Requires libldns 1.7.0 or 1.7.1 (#85)
+ - Requires OpenSSL 1.1.1 unless Ed25519 (algorithm 15)
+   has been disabled
+
+ [Changed]
```
Comment (mattias-p) [inline Changes]: This is an addition, not a change. Please read https://semver.org/ and https://keepachangelog.com/en/1.0.0/.

File: Changes (hunk 3 above)
Comment (mattias-p) [reply Changes]: I'm not sure who you're agreeing with.

**Comment by matsduf** [comment]
@mattias-p and @sandoche2k, please rereview.

File: Changes (hunk 3 above)
Comment (sandoche2k) [reply Changes]: I agree with @mattias-p

File: Changes (hunk 2 above)
Comment (sandoche2k) [reply Changes]: I seem to agree with @matsduf. If we go by @mattias-p logic, then all changes file should be re-written with same option?

File: Changes (hunk 3 above)
Comment (matsduf) [reply Changes]: My approach has been to include all changes to the repository, and I do not see the problem of including things concerning Travis, and in that way have it documented in the Changes file, but on your request I will exclude those. This is a change from earlier behavior, e.g. see the same file under version 2.0.0 a year ago.

I will update and remove travis from the upcoming release.

File: Changes (hunk 2 above)
Comment (matsduf) [reply Changes]: I think it would be good if we defined a template for the Changes files. It should not be too complex so that it is hard to use.

**Comment by matsduf** [comment]
@mattias-p and @sandoche2k, please re-review.

**Comment by matsduf** [comment]
@sandoche2k and @mattias-p, please review!

**Pull request merged**